src/python/install.sh

```diff
@@ -315,7 +315,7 @@ sudo_if() {
 
 install_user_package() {
     PACKAGE="$1"
-    sudo_if "$INSTALL_PATH/bin/python3" -m pip install --user --upgrade --no-cache-dir "$PACKAGE"
+    sudo_if "$CURRENT_PATH/bin/python" -m pip install --upgrade --no-cache-dir "$PACKAGE"
```

Comment:
How do we ensure `CURRENT_PATH` is set properly when this function executes? It looks like if `"${PYTHON_VERSION}" != "none"`, `CURRENT_PATH` won't be set when jupyterlab tries to install.

If it's not a supported configuration to pass `none` for python version but also to request jupyterlab installation, then we should probably error out above and say the combination of options specified is not valid.

Reply:
Good point. let me address that

Reply:
Commented above, but please use `python3` for the command.

Also - be sure to add an export statement to the script to get the version of python that is installed into the PATH.  That won't happen by default.  Furthermore, if you fire this where it is now in the script, it will pick up these env vars: 

https://github.com/devcontainers/features/blob/main/src/python/install.sh#L422-L423

You should `unset` these first.